source/includes/_intro.md

```diff
@@ -44,9 +44,11 @@ Error Code | Reason | Description
 ---------- | ------- | -------
 400 | Bad Request | Your request is malformed or missing mandatory data
 401 | Unauthorized | Your API key is either wrong, missing or you have no permissions for such request
+403 | Forbidden | Your request is trying to access data it has no credentials for
 404 | Not Found | The specified resource could not be found
-500 | Internal Server Error | We had a problem with our server. Try again later.
-503 | Service Unavailable | We're temporarily offline for maintenance. Please try again later.
+409 | Conflict | There was a conflict with processing your request. Try later or change the data being submitted
+500 | Internal Server Error | We had a problem with our server. Try again later
```

Comment:
from `We had a problem with our server.` to `We are having problems with our server`